docs/maintainer-playbook.md

```diff
@@ -1,3 +1,21 @@
 ## Maintainer playbook
 
+### Tagging Shadow releases
+
+The following commands can be used to tag a new version of Shadow, after which an
+archive will be available on github's [releases page](https://github.com/shadow/shadow/releases).
+
+```bash
+git checkout master
+git tag -s v1.10.0
+git push origin v1.10.0
+```
```

Comment:
I recently started using the python module `bumpversion` for automatically bumping the version number. We could also make it automatically tag the release. (See the `.bumpversion.cfg` file in the root.)

Should we continue to use bumpversion?

It might still be good to keep the above commands as an alternative method, but if we don't use bumpversion we'll have to update the version number in the toplevel CMakeLists.txt file manually.

Reply:
I haven't used the bumpversion tool before, but poked at it a bit and took a stab at updating the instructions.

I think it makes sense to use the --commit flag with bumpversion to automatically generate a commit. I see we could also change the config to make that the default, but I think I prefer having to explicitly ask for it on the command-line.

bumpversion also has a -t option to automatically tag the new version, but it wouldn't be a signed tag. Assuming that matters (does it?), I left the instructions to create a signed tag instead. Unfortunately that means a slightly cludgey bit of shell script to grab the current version number. Surprisingly bumpversion doesn't seem to have an option to just grab and print the current version string.